applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_interface_element.cpp

```diff
@@ -374,10 +374,11 @@ void MockElementWithTotalStressVectors::SetIntegrationMethod(IntegrationMethod C
     mOptionalCustomIntegrationMethod = CustomIntegrationMethod;
 }
 
-auto MakeElementGlobalPtrContainerWith(const auto& rpElement)
+template <typename DerivedElementPtrType>
+GlobalPointersVector<Element> MakeElementGlobalPtrContainerWith(const DerivedElementPtrType& rpElement)
 {
     auto p_element = Element::Pointer{rpElement};
-    return GlobalPointersVector{{GlobalPointer{p_element}}};
+    return {{GlobalPointer<Element>{p_element}}};
```

Comment:
Might be me, but it seems to me we have one set of curly braces too much. Feel free to fix this later if the build already passes
```suggestion
    return {GlobalPointer<Element>{p_element}};
```

Reply:
I'm not sure. The outer curly braces are used for constructing the `GlobalPointersVector<Element>` container (i.e. the result), whereas the inner ones are used for constructing a `std::initializer_list<GlobalPointer<Element>>`, which is passed to the constructor of the container. Or am I overlooking something?

Reply:
I'm probably simplifying this too much in my head. At least for std::vector, I'm used to just return {<object>}, i.e.
```
std::vector<int> createVector()
{
   return {1, 2, 3, 4, 5}
}
```
But probably it works differently for a vector or simpler internal types (probably there are some implicit conversions going on). 

I'd expect the curly braces around `p_element` are already enough to create the GlobalPointer<Element> and then one pair of braces to create the GlobalPointersVector, but I might be wrong.

Reply:
Yes, now that I see your counterexample I believe that you are right. Let's change it in a separate PR.